server/src/main/java/io/unitycatalog/server/persist/CommitRepository.java

```diff
@@ -0,0 +1,230 @@
+package io.unitycatalog.server.persist;
+
+import io.unitycatalog.server.exception.BaseException;
+import io.unitycatalog.server.exception.ErrorCode;
+import io.unitycatalog.server.model.ColumnInfos;
+import io.unitycatalog.server.model.Commit;
+import io.unitycatalog.server.model.CommitInfo;
+import io.unitycatalog.server.model.CommitMetadataProperties;
+import io.unitycatalog.server.model.DataSourceFormat;
+import io.unitycatalog.server.model.Metadata;
+import io.unitycatalog.server.model.TableType;
+import io.unitycatalog.server.persist.dao.CommitDAO;
+import io.unitycatalog.server.persist.dao.TableInfoDAO;
+import io.unitycatalog.server.persist.utils.FileOperations;
+import io.unitycatalog.server.persist.utils.TransactionManager;
+import io.unitycatalog.server.utils.ServerProperties;
+import io.unitycatalog.server.utils.ValidationUtils;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.UUID;
+import org.hibernate.Session;
+import org.hibernate.SessionFactory;
+import org.hibernate.query.Query;
+
+/** Repository for managing coordinated commits for managed Delta tables in Unity Catalog. */
+public class CommitRepository {
```

Comment:
can you name these specifically with Delta? 
these are related to Delta tables only.. lets make that explicit in all these new classes.